crunchy-cli-core/src/archive/command.rs

```diff
@@ -64,6 +64,10 @@ pub struct Archive {
     #[arg(long)]
     pub(crate) output_specials: Option<String>,
 
+    #[arg(help = "Sanitize file names for all operating systems")]
+    #[arg(long, default_value_t = false)]
+    pub(crate) universal_filenames: bool,
```

Comment:
Renaming the flag (and help) to `universal_output` would be better to keep name consistency with the `output` and `output-specials` flag.
Also, it should be clarified that this flag only affects template options and not static characters.